torch/_decomp/decompositions.py

```diff
@@ -4020,6 +4020,31 @@ def scaled_dot_product_flash_attention(
     )
 
 
+@register_decomposition(aten.hann_window.periodic)
+def hann_window_periodic(
```

Comment:
What is the reason why we’d want to put this in refs?

Reply:
Refs are tested far more thoroughly by creating an `PythonRefInfo` to run the entire suite of opinfo tests against it. The only reason to make a decomp instead of a ref is if you couldn't implement it as a ref, e.g. because it's not in the public API.

Reply:
@SherlockNoMad thoughts?